src/zabapgit_page_stage.prog.abap

```diff
@@ -25,8 +26,7 @@ CLASS lcl_gui_page_stage DEFINITION FINAL INHERITING FROM lcl_gui_page.
   PRIVATE SECTION.
     DATA: mo_repo  TYPE REF TO lcl_repo_online,
           ms_files TYPE ty_stage_files,
-          mo_stage TYPE REF TO lcl_stage,
-          mv_ts    TYPE timestamp.
+          mv_seed  TYPE string.
```

Comment:
added at variable: `" Unique page id to bind JS sessionStorage`
at constructor: `" Generate based on time unless obtained from diff page`
and also some comment to diff_page (where it is also used).

Does it look OK ?